api_tests/preprints/views/test_preprint_detail.py

```diff
@@ -22,10 +22,11 @@
 from reviews.workflow import States
 
 
-def build_preprint_update_payload(node_id, attributes=None, relationships=None):
+def build_preprint_update_payload(node_id, attributes=None, relationships=None, type='POST'):
```

Comment:
The types here should be the content type (`nodes`, `users`, `preprints`, etc). If these tests don't fail with the current data, then please ensure that the type that you're using matches the type of the object as used by the API.

Reply:
And also please make an explicit test where you put in no data it returns your error, and one where you put the wrong type for the endpoint and it returns an error (`409 Conflict` in the case of the wrong type)